src/OpenTelemetry.Api/Context/Propagation/ITextFormatActivity.cs

```diff
@@ -49,5 +49,14 @@ public interface ITextFormatActivity
         /// <param name="getter">Function that will return string value of a key with the specified name.</param>
         /// <returns>Activity context from it's text representation.</returns>
         ActivityContext Extract<T>(T carrier, Func<T, string, IEnumerable<string>> getter);
+
+        /// <summary>
+        /// Tests if an activity context has been injected into a carrier.
+        /// </summary>
+        /// <typeparam name="T">Type of object to extract context from. Typically HttpRequest or similar.</typeparam>
+        /// <param name="carrier">Object to extract context from. Instance of this object will be passed to the getter.</param>
+        /// <param name="getter">Function that will return string value of a key with the specified name.</param>
+        /// <returns><see langword="true" /> if the carrier has been injected with an activity context.</returns>
+        bool IsInjected<T>(T carrier, Func<T, string, IEnumerable<string>> getter);
```

Comment:
can `Fields` collection be used here? I'm fine having this method, I suggest to describe use case in a form of an issue on specification repo explaining the need for this.

Reply:
@SergeyKanzhelev I don't think `Fields` will work alone to determine if a request is instrumented or not. The `TraceContextFormatActivity` for example has 2 `Fields` defined: `traceparent` and `tracestate`. A request is instrumented if it has `traceparent`, `tracestate` is optional. So not enough information in `Fields` for me to know which ones are required to make the determination.

Reply:
Also, spec issue open: https://github.com/open-telemetry/opentelemetry-specification/issues/674